prog/weave-kube/weave-monitoring-svc.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+   name: weave-net-monitoring
+   namespace: kube-system
+   annotations:
+     prometheus.io.scrape: "true"
+spec:
+   selector:
+      name: weave-net
```

Comment:
Changed the name of each service `weave-{net,npc}-monitoring` -> `weave-monitoring-{net,npc}`). Also, pinged the weave-net team about using the kubelint script.